libnmstate/iplib.py

```diff
@@ -14,12 +14,85 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+import ipaddress
+import logging
+import re
+import six
 
 _IPV6_LINK_LOCAL_NETWORK_PREFIXES = ['fe8', 'fe9', 'fea', 'feb']
 _IPV6_LINK_LOCAL_NETWORK_PREFIX_LENGTH = 10
 
+IPV4_LINK_LOCAL_NETWORK = "169.254.0.0/16"
+IPV6_LINK_LOCAL_NETWORK = "fe80::/10"
+IPV6_KERNEL_AUTO_MULTICAST_NETWORK = 'ff00::/8'
+
+MAIN_ROUTE_TABLE_NAME = 'main'
+MAIN_ROUTE_TABLE_ID = 0
+
 
 def is_ipv6_link_local_addr(ip, prefix):
     return (ip[:len(_IPV6_LINK_LOCAL_NETWORK_PREFIXES[0])]
             in _IPV6_LINK_LOCAL_NETWORK_PREFIXES and
             prefix >= _IPV6_LINK_LOCAL_NETWORK_PREFIX_LENGTH)
+
+
+def _is_subnet_of(a, b):
```

Comment:
Please keep the caller below the callee so the details will always be below.
This on need to be placed below `is_subnet_of`.

Reply:
Done.